ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/ConsumerGroupsTable.tsx

```diff
@@ -76,31 +75,17 @@ export function ConsumerGroupsTable({
   filterName: string | undefined;
   filterState: ConsumerGroupState[] | undefined;
   consumerGroups: ConsumerGroup[] | undefined;
-  refresh: (() => Promise<ConsumerGroup[]>) | undefined;
   onFilterNameChange: (name: string | undefined) => void;
-  onFilterStateChange: (status: ConsumerGroupState[] | undefined) => void;
+  onFilterStateChange: (state: ConsumerGroupState[] | undefined) => void;
   onResetOffset: (consumerGroup: ConsumerGroup) => void;
 } & Pick<
   TableViewProps<ConsumerGroup, (typeof ConsumerGroupColumns)[number]>,
   "isColumnSortable" | "onPageChange" | "onClearAllFilters"
 >) {
   const t = useTranslations();
-  const [consumerGroups, setConsumerGroups] = useState(initialData);
-  useEffect(() => {
-    let interval: ReturnType<typeof setInterval>;
-    if (refresh) {
-      interval = setInterval(async () => {
-        const consumerGroups = await refresh();
-        if (consumerGroups) {
-          setConsumerGroups(consumerGroups);
-        }
-      }, 5000);
-    }
-    return () => clearInterval(interval);
-  }, [refresh]);
   return (
     <TableView
-      itemCount={consumerGroups?.length}
```

Comment:
@hemahg , I didn't notice earlier, but I think this line change disables pagination.

Reply:
pagination will be applied if the number of records is more than 20

Reply:
Ok, got it. Somehow it's not working with the storybook for this page. See the changes in chromatic for the build: https://www.chromatic.com/build?appId=66cf23a2f89a8283f775d05b&number=328